client/src/app/app.component.scss

```diff
@@ -0,0 +1,37 @@
+// Standard Styles
+body {
+    /* Margin bottom by footer height */
+    margin-bottom: 60px;
+  }
+  .footer {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    /* Set the fixed height of the footer here */
+    height: 60px;
```

Comment:
In the design document, the `footer` element height property is defined to be `10%` of the page height. Why have you defined it here with a hard-coded height of `60px`?